content/docs/functions/json_build_object.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Postgres json_build_object function
+subtitle: "Builds a JSON object out of a variadic argument list."
+enableTableOfContents: true
+---
+
+
+Use `json_build_object` to construct a JSON object from a set of key-value pairs, creating a JSON representation of a row or set of rows.
+
+
+Function signature:
+
+
+```sql
+json_build_object ( VARIADIC "any" ) → json
+```
+
+
+## `json_build_object` example
+
+
+Let's consider a scenario where we have a table storing information about users:
+
+
+**users**
+
+```text
+| id |   name   | age |   city  
```

Comment:
Content in rows 3 and 4 is duplicated. I'm not sure why.

Reply:
yeah this was a typo, deleted rows 3 and 4